packages/plugins/mention/src/ui/MentionRender.tsx

```diff
@@ -1,29 +1,101 @@
-import { PluginElementRenderProps } from '@yoopta/editor';
-import { useSelected } from 'slate-react';
+import React, { useState, useCallback, useEffect } from 'react';
+import { PluginElementRenderProps, generateId } from '@yoopta/editor';
+import { useSlate, useSelected } from 'slate-react';
+import { Transforms, Editor } from 'slate';
 
-const MentionRender = (props: PluginElementRenderProps) => {
-  const { url, target, rel, character } = props.element.props || {};
+const MENTION_TRIGGER = '@';
+
+interface MentionRenderProps extends PluginElementRenderProps {
+  initialMentions: string[];
+  fetchMentions: (query: string) => Promise<string[]>;
```

Comment:
Should we give the possibility to override this ? I might want to group mentions